build_tools/benchmarks/suites/cmake_rule_generator.py

```diff
@@ -0,0 +1,204 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Generator that generates CMake rules from python defined benchmarks.
+
+The rules will build required artifacts to run benchmarks.
+"""
+
+from dataclasses import dataclass
+from typing import List, Optional
+import os
+import pathlib
+import string
+import urllib.parse
+
+from .definitions import common_definitions, iree_definitions
+from . import iree_benchmarks
+
+# Template dir: build_tools/benchmarks/suites/../../cmake
+TEMPLATE_DIR = pathlib.Path(__file__).parents[2] / "cmake"
```

Comment:
nit: This relative path handling is brittle and difficult to spot when refactoring. Starting from the repo root or just including the files in this directory would be more robust.

Reply:
Moved to the same directory.